src/compas/geometry/triangulation_earclip.py

```diff
@@ -1,8 +1,248 @@
-from compas.geometry import is_ccw_xy, is_point_in_triangle_xy
+class Ear:
```

Comment:
python 2.x old-style classes compatibility:

```suggestion
class Ear(object):
```